src/frontend/src/views/Data/components/SamplesView/components/SamplesTable/components/LineageTooltip/components/GeneralViralLineageTooltip/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { Tooltip } from "czifui";
+import { LineageTooltipProps } from "../../lineageTooltipConfig";
+import { Label, Text, Wrapper } from "./style";
+
+const KEY_TO_LABELS = {
+  lineage: "Lineage",
+  qc_status: "QC Status",
+  scorpio_call: "Scorpio Call",
+  scorpio_support: "Scorpio Support",
+  lineage_software_version: "Version",
+  lineage_type: "Lineage Type",
+  lineage_probability: "Lineage Probability",
+  last_updated: "Last Updated",
+  reference_dataset_name: "Nextclade Dataset",
```

Comment:
- [ ]  Can someone please confirm that this is the correct field to use for Nextclade dataset?

Reply:
Looks right to me - see https://github.com/chanzuckerberg/czgenepi/blob/c1370202b18134631696ac1ba52cc6eb17bcbad4/src/backend/aspen/database/models/lineages.py#L113...L121

Reply:
thanks!